storage/mysql/mysql.go

```diff
@@ -70,6 +71,9 @@ func New(ctx context.Context, db *sql.DB, opts ...func(*tessera.StorageOptions))
 		klog.Errorf("Failed to ping database: %v", err)
 		return nil, err
 	}
+	if s.newCheckpoint == nil || s.parseCheckpoint == nil {
```

Comment:
Is this a required option here because this storage implementation doesn't track the tree size separately from the stored `checkpoint`?

Probably should say that this is a requirement for _this_ storage impl on the c'tor comment.

Reply:
In the current MySQL implementation, the `tessera.WithCheckpointSignerVerifier` option is mandatory. I've added comments to the `mysql.New()` method.

Is there any scenario that doesn't require the signer and verifier? Let's say we mirror the log. Is it necessary to sign the checkpoint with another pairs of keys?

Reply:
Yeah, that's one scenario - for mirrors we don't have access to the source's private key, but I can't imagine we'd want to sign with a separate key.

Reply:
Gotcha. I've updated the logic to make `tessera.WithCheckpointSignerVerifier` opt optional in `mysql.New()`.